bitnami/external-dns/README.md

```diff
@@ -170,7 +170,8 @@ The following table lists the configurable parameters of the external-dns chart
 | `service.loadBalancerSourceRanges`  | List of IP CIDRs allowed access to load balancer (if supported)                                          | `[]`                                                        |
 | `service.annotations`               | Annotations to add to service                                                                            | `{}`                                                        |
 | `rbac.create`                       | Weather to create & use RBAC resources or not                                                            | `true`                                                      |
-| `rbac.serviceAccountName`           | ServiceAccount (ignored if rbac.create == true)                                                          | `default`                                                   |
+| `rbac.serviceAccount.create`        | If true and rbac.create is also true, a service account will be created                                  | `true`                                                      |
+| `rbac.serviceAccount.name`          | existing ServiceAccount to use (ignored if rbac.create=true and rbac.serviceAccount.create=true)         | `default`                                                   |
```

Comment:
Hi @cw-sakamoto 

Please note that you're modifying the values structure. Therefore, that would break backwards compatibility unless people adapt their values.yaml. We usually request to bump the major version for every change of these characteristics.

On the other hand, I don't think it's necessary to break this, you can use a different approach:

| Parameter | Description  | Default   |
| -----------| ------------| --------- |
| `rbac. serviceAccountName ` | ServiceAccount to use. A name is generated using the `external-dns.fullname` template if it is not set and `rbac.create` is `true` | `nil` |

Then, adapt the PR to ensure the approach previously defined is true.

Reply:
@juan131 Thanks for your comments!
As you say, maintaining backward compatibility is important, so I fixed it.
https://github.com/bitnami/charts/pull/2088/commits/0ec36b5b5c7f826f5cc5ee1be9df9e60d32e0805

I think that the charts ​​will work maintaining backward compatible, how about?